cmd/nfd-master/main.go

```diff
@@ -122,6 +122,8 @@ func initFlags(flagset *flag.FlagSet) (*master.Args, *master.ConfigOverrideArgs)
 			"in the same format as in the config file (i.e. json or yaml). These options")
 	flagset.BoolVar(&args.EnableLeaderElection, "enable-leader-election", false,
 		"Enables a leader election. Enable this when running more than one replica on nfd master.")
+	flagset.Int64Var(&args.ListSize, "informer-list-size", 200,
```

Comment:
Just pondering (bike-shedding) on the naming. Would `-informer-paginate` or `-informer-page-size` be more descriptive, thoughts?

Reply:
Yes I like that, I'll update